pkg/reconciler/revision/resources/deploy.go

```diff
@@ -99,6 +106,23 @@ func makePodSpec(rev *v1.Revision, loggingConfig *logging.Config, tracingConfig
 
 	podSpec := BuildPodSpec(rev, append(BuildUserContainers(rev), *queueContainer))
 
+	if features.RuntimeV1VarLogVolume == config.Enabled {
+		podSpec.Volumes = append(podSpec.Volumes, varLogVolume)
+
+		for i, container := range podSpec.Containers {
+			if container.Name == QueueContainerName {
+				continue
+			}
+
+			varLogMount := varLogVolumeMount.DeepCopy()
+			varLogMount.SubPathExpr += container.Name
+			container.VolumeMounts = append(container.VolumeMounts, *varLogMount)
+			container.Env = append(container.Env, buildVarLogSubpathEnvs()...)
+
+			podSpec.Containers[i] = container
```

Comment:
Something that I'm not sure we've thought about enough is that we are using the same `/var/log` mount for all of the containers in a multi-container deployment.

cc @skonto @markusthoemmes

Reply:
The subpath expression gives them unique directories that aren't visible to each other